docs: update ceph monitoring examples

galexrt opened the pull request:
**Description of your changes:**

This updates the Ceph docs regarding the Grafana dashboards new version
requirement.

Signed-off-by: Alexander Trost <galexrt@googlemail.com>

**Which issue is resolved by this Pull Request:**
Resolves #6430

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [x] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [x] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [x] Documentation has been updated, if necessary.
- [x] Unit tests have been added, if necessary.
- [x] Integration tests have been added, if necessary.
- [x] Pending release notes updated with breaking and/or notable changes, if necessary.
- [x] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [x] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

leseb reviewed (changes requested):
Actually, can you split into 2 commits since you changed 2 things? Thanks!

galexrt commented:
@leseb I have removed the second change and adjusted the commit message.

I talked with some Prometheus folks and they adviced to not use the relabeling there but to change the dashboards, as of now the dashboards have been updated to hopefully be more resilient.

yanchicago commented:
Grafana 7.2.0 was released 9/23. The version I used was released 1/27. Any reason that it has to be on the exact latest release? Especially, the rook release 1.4 was released before Grafana 7.2?

galexrt commented:
@yanchicago The Grafana I have used to update the dashboards is running that version. The dashboards might also work with Grafana `>= 7.0.0`, but I can't guarantee that that is why the `7.2.0` is named as a requirement.

The Rook version is not coupled with the dashboards. The Ceph MGR Prometheus module would be the one the dashboards would need to be coupled with, in regards to the version the dashboards are compatible with which module versions.

yanchicago commented:
@galexrt My understanding is that Ceph only provides Prometheus exporter function. In my case I have promethues 2.16 which should work with Grafana 6.6.2. If not, which prometheus version should work with ceph 15.x?
